db/event_listener.go

```diff
@@ -1,15 +1,17 @@
 package db
 
+import "time"
+
 type EventListener interface {
-	OnIO(write bool)
+	OnIO(write bool, took time.Duration)
```

Comment:
`duration` would be better name instead of `took`

Reply:
done